django/utils/feedgenerator.py

```diff
@@ -223,7 +223,7 @@ def latest_post_date(self):
                     if latest_date is None or item_date > latest_date:
                         latest_date = item_date
 
-        return latest_date or datetime.datetime.now()
+        return latest_date or datetime.datetime.now(tz=timezone.utc)
```

Comment:
Oh, I missed that comment, thanks! In that case the current code looks good to me.

I don't think performance of latest_post_date is too important, but I will leave the following comment from jango/utils/timezone.py:

```py
# timeit shows that datetime.now(tz=utc) is 24% slower
return datetime.utcnow().replace(tzinfo=utc)
```